packages/backend/src/config/getEnv.ts

```diff
@@ -49,3 +49,7 @@ getEnv.boolean = function getBooleanFromEnv(
   }
   throw new Error(`Missing environment variable ${name}!`)
 }
+
+getEnv.featureFlag = function getFeatureFlagFromEnv(name: string): boolean {
+  return getEnv.boolean(name, false)
+}
```

Comment:
With how simple the function is is it really necessary?

Reply:
WITH 80 CHARS PER LINE LIMIT I AM FIGHTING FOR EACH CHARACTER TO MAKE THE CONFIG READABLE xD but seriously i also thought about dropping it... but then there are new lines everywhere.